mappings/net/minecraft/CoordComparator.mapping

```diff
@@ -0,0 +1,14 @@
+CLASS net/minecraft/class_2349 net/minecraft/CoordComparator
+	COMMENT TODO: Use better name
+	FIELD field_10572 x F
+	FIELD field_10573 y F
+	FIELD field_10574 z F
+	FIELD field_10575 buffer [I
+	METHOD <init> ([IFFF)V
+		ARG 1 buffer
```

Comment:
Why is this code taking a buffer in to compare Coordinates?

Reply:
i have no idea, but it takes a int array of size 3 together with xyz float components

Reply:
i'll mark this as resolved, but feel free to ask again if you still have dubts about the name